man/systemd-firstboot.xml

```diff
@@ -238,6 +238,13 @@
         option should not be used lightly.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--root-password-is-hashed</option></term>
+
+        <listitem><para>Indicates that the root password specified with <option>--root-password</option> is
+        already hashed.</para></listitem>
+      </varlistentry>
```

Comment:
hmm, not a fan of having options that change what other options take. wouldn't it be simpler if we had --root-password= for unhashed pwds, and --root-password-hashed= for hashed passwords? i.e. instead of making this a string argument and a bool argument just make it two string options?

btw, we generally suffix parameters that take arguments with "=" in log messages and documentation. Hence, please write the option name as `<option>--root-password=</option>`

Reply:
Replaced with `--root-password-hashed=` but kept the implementation the same. I followed the existing behaviour of overriding earlier `--root-password` and `--root-password-file` options instead of erroring when more than one of the root password options is specified. Shouldn't we make that an error though (would break backward compat though)?